model/task/task.go

```diff
@@ -993,3 +996,100 @@ func (t *Task) MergeNewTestResults() error {
 	}
 	return nil
 }
+
+func FindRunnable() ([]Task, error) {
+	expectedStatuses := [3]string{evergreen.TaskSucceeded, evergreen.TaskFailed, ""}
+
+	matchActivatedUndispatchedTasks := bson.M{
```

Comment:
I would just declare this as a []bson.M{} (but I made this arguement to brian and he liked this form better for reasons I still don't understand, so it's fair to do it either way.)